theme/lynx-ui-home/index.scss

```
@@ -3,222 +3,257 @@
-    .before-title {
-      display: flex;
-      padding: 0.5rem;
-      /* 8px */
-      font-size: 0.875rem;
-      /* 14px */
-      line-height: 1.25rem;
-      /* 20px */
-      background-color: hsl(var(--background));
+  .startBuildingTitle {
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)